### ISSUE TITLE
Accessibility: Not focussing on the header when tabbing into a page 

### ISSUE THREAD

**Issue by JT-39** [dfe-analytical-services/local-authority-interactive-tool#59]
**Improvement Overview**

When tabbing onto a page it should focus onto the header.

**Current Issue**

When tabbing onto a page it just says blank. Instead, it should focus on the header. This is across all pages.

**Proposed Enhancement**

A review of the code used to setup the pages should be done to check if this is causing the issue. Most pages do not use the `shinyGovstyle::gov_main_layout()` function, so maybe the `bslib::nav_panel()` or other such functions do not support this functionality yet and causing the issue. Could also be due to the header being dynamic for the main pages of the app.

**Impact**

This should improve the experience for screen readers.

**Additional Information**
